### DISCUSSION TITLE
Allow specifying the default domain for unbounded integer variables in flatzinc interface

### DISCUSSION THREAD

**Discussion by CervEdin** [google/or-tools#4411] [Feature requests] [answered]
If you write a minizinc model with 4095+ `var int` the model will be declared invalid 
```
array[1..4096] of var int: a;
```
because the domain defaults to `-(2^50)..(2^50)`
https://github.com/google/or-tools/blob/ed94162b910fa58896db99191378d3b71a5313af/ortools/flatzinc/cp_model_fz_solver.cc#L45C1-L47C1
and this will fail the model overflow check.

It would be useful to allow users to adjust this default, say by passing a CLI parameter to override this default value.
I guess the canonical way of doing this is to add another  `extraFlags`
https://github.com/google/or-tools/blob/ed94162b910fa58896db99191378d3b71a5313af/ortools/flatzinc/cp-sat.msc.in#L11

so something like
```
"extraFlags": [
    ["--params", "Provide parameters interpreted as a text SatParameters proto", "string", ""],
    ["--unbounded-int-var-domain", "Domain of unbounded int variables", "int", ""]
  ],
```

**Comment by lperron** [comment]
already there: 
ABSL_FLAG(int64_t, fz_int_max, int64_t{1} << 50,
          "Default max value for unbounded integer variables.");

**Comment by lperron** [comment, marked as answer]
Implemented
